infra/order-generator/index.ts

```diff
@@ -29,80 +31,129 @@ export = () => {
   const vpcId = baseStack.getOutput('VPC_ID') as pulumi.Output<string>;
   const privateSubnetIds = baseStack.getOutput('PRIVATE_SUBNET_IDS') as pulumi.Output<string[]>;
   const boundlessAlertsTopicArn = baseConfig.get('SLACK_ALERTS_TOPIC_ARN');
+  const interval = baseConfig.require('INTERVAL');
+  const lockStake = baseConfig.require('LOCK_STAKE');
+  const rampUp = baseConfig.require('RAMP_UP');
+  const minPricePerMCycle = baseConfig.require('MIN_PRICE_PER_MCYCLE');
+  const maxPricePerMCycle = baseConfig.require('MAX_PRICE_PER_MCYCLE');
+  const secondsPerMCycle = baseConfig.require('SECONDS_PER_MCYCLE');
   
-  const simpleConfig = new pulumi.Config("order-generator-simple");
-  const simplePrivateKey = isDev ? pulumi.output(getEnvVar("SIMPLE_PRIVATE_KEY")) : simpleConfig.requireSecret('PRIVATE_KEY');
-  const simpleInterval = simpleConfig.require('INTERVAL');
-  const simpleLockStake = simpleConfig.require('LOCK_STAKE');
-  const simpleRampUp = simpleConfig.require('RAMP_UP');
-  const simpleMinPricePerMCycle = simpleConfig.require('MIN_PRICE_PER_MCYCLE');
-  const simpleMaxPricePerMCycle = simpleConfig.require('MAX_PRICE_PER_MCYCLE');
+  const imageName = getServiceNameV1(stackName, `order-generator`);
+  const repo = new awsx.ecr.Repository(`${imageName}-repo`, {
```

Comment:
I've moved this out of the component so that we build and maintain only one image instead of one per component